common/src/backends/headers.rs

```diff
@@ -33,15 +33,15 @@ impl Header for XShuttleAdminSecret {
     }
 }
 
-pub static X_SHUTTLE_ACCOUNT_NAME: HeaderName = HeaderName::from_static("x-shuttle-account-name");
+pub static X_SHUTTLE_USER_ID: HeaderName = HeaderName::from_static("x-shuttle-user-id");
```

Comment:
Changing the header name will break old deployers iirc. So we can't do that

Reply:
This one is only used between auth and gateway, so we can

Reply:
Fixed in 1edf4073